### DISCUSSION TITLE
Welcome to dns-updater Discussions!

### DISCUSSION THREAD

**Discussion by alexplesoiu** [alexplesoiu/dns-updater#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SpasovskiFilip** [comment]
Hi Alex, great idea.

My name is Filip, I work as an automation test engineer, and this is mostly a hobby for me.
I recently got a Synology NAS, and started experimenting with reverse proxy and domains. I was wondering how to update my IP if it changed, and after some digging I found your code. With a little bit of modification, worked wonders for me 😸.

I'm not an expert in neither python nor docker, but I'm learning and I'd like to contribute as much as I can.

**Reply by alexplesoiu** [reply]
Hello Filip,

Don't worry, this project is pretty simple and you don't need to be an expert to add your contribution to it. At least, I don't care if you are an expert or a junior, every contribution, if it's good, will be helpful! :)
